Circle.h

```diff
@@ -7,60 +11,70 @@
 using namespace std;
 
 /// <summary>
-/// ����� ����������
+/// Класс окружности
 /// </summary>
 class Circle
 {
 protected:
-	/// <summary>
-	/// ���������� ������ �� ��� �������
-	/// </summary>
-	double x;
-	/// <summary>
-	/// ���������� ������ �� ��� �������
-	/// </summary>
-	double y;
-	/// <summary>
-	/// ������
-	/// </summary>
-	double radius;
+    /// <summary>
+    /// Координата центра по оси абсцисс
+    /// </summary>
+    double x;
+    /// <summary>
+    /// Координата центра по оси ординат
+    /// </summary>
+    double y;
+    /// <summary>
+    /// Радиус
+    /// </summary>
+    double radius;
 public:
-	/// <summary>
-	/// ����������� �� ���������
-	/// </summary>
-	Circle();
+    /// <summary>
+    /// Конструктор по умолчанию
+    /// </summary>
+    Circle();
+
+    /// <summary>
+    /// Конструктор с параметрами
+    /// </summary>
+    /// <param name="_x"></param>
+    /// <param name="_y"></param>
+    /// <param name="_radius"></param>
+    Circle(double _x, double _y, double _radius);
+
+    /// <summary>
+    /// Circle destructor
+    /// </summary>
+    ~Circle();
 
-	/// <summary>
-	/// ����������� � �����������
-	/// </summary>
-	/// <param name="_x"></param>
-	/// <param name="_y"></param>
-	/// <param name="_radius"></param>
-	Circle(double _x, double _y, double _radius);
+    /// <summary>
+    /// Виртуальный (может быть переопределен)
+    /// метод нахождения площади окружности
+    /// </summary>
+    /// <returns></returns>
+    virtual double get_square();
 
-	/// <summary>
-	/// ����������� (����� ���� �������������) 
-	/// ����� ���������� ������� ����������
-	/// </summary>
-	/// <returns></returns>
-	virtual double get_square();
+    /// <summary>
+    /// Метод нахождения периметра окружности
+    /// </summary>
+    /// <returns></returns>
+    double get_perimeter();
 
-	/// <summary>
-	/// ����� ���������� ��������� ����������
-	/// </summary>
-	/// <returns></returns>
-	double get_perimeter();
+    /// <summary>
+    /// Метод перемещения окружности
+    /// </summary>
+    /// <param name="dx"></param>
+    /// <param name="dy"></param>
+    void move(double dx, double dy);

```

Comment:
Если параметры функции не изменяются, то они явно должны быть обозначены `const`
[By Alex Allain](https://www.cprogramming.com/tutorial/const_correctness.html)
>The const keyword allows you to specify whether or not a variable is modifiable. You can use const to prevent modifications to variables and const pointers and const references prevent changing the data pointed to (or referenced).
>Const дает вам возможность более четко документировать вашу программу и фактически обеспечивать соблюдение этой документации. Ключевое слово const обеспечивает вашим пользователям гарантии, позволяющие оптимизировать производительность без угрозы повреждения их данных, обеспечивая соблюдение вашей документации. Например, константные ссылки позволяют указать, что упомянутые данные не будут изменены; это означает, что вы можете использовать константные ссылки как простой и немедленный способ повышения производительности для любой функции, которая в настоящее время принимает объекты по значению, не беспокоясь о том, что ваша функция может изменить данные. Даже если это произойдет, компилятор предотвратит компиляцию кода и предупредит вас о проблеме. С другой стороны, если бы вы не использовали константные ссылки, у вас не было бы простого способа гарантировать, что ваши данные не были изменены.